### PR TITLE
retrained model; global variable fixes in flask_api.py

### PR DESCRIPTION
**Pull request by cameron-toy**
## What's New?
- Retrained model
- Fixed some bugs in flask_api.py
- Added `initialize*()` functions to flask_api.py; see note in source

## Type of change (pick-one)
- [x] Bug fix (_non-breaking change which fixes an issue_)
- [ ] New feature (_non-breaking change which adds functionality_)
- [ ] Breaking change (_fix or feature that would cause existing functionality to not work as expected_)
- [ ] This change requires a documentation update

### CODE REVIEW

File: nimbus_nlp/question_classifier.py [hunk 1]
```diff
@@ -12,97 +12,6 @@
 class QuestionClassifier:
     def __init__(self, file_path="question_set_clean.csv"):
         self.classifier = None
```
Comment (richagadgil) [inline nimbus_nlp/question_classifier.py:14]: Did you delete all this stuff from question_classifier?

**Review by richagadgil — APPROVED** [review]

File: nimbus_nlp/question_classifier.py (hunk 1 above)
Comment (cameron-toy) [reply nimbus_nlp/question_classifier.py:14]: Yes. The last PR re added the old feature extraction functions and I removed those. The other functions are just moved within the class. You can check the full text of the file to see no functionality had been removed.

**Review by chidiewenike — APPROVED** [review]

**Review by Jason-Ku — APPROVED** [review]

File: Entity/Profs.py [hunk 2]
```diff
@@ -7,11 +7,11 @@
 
 class Profs(Base):
     __tablename__ = "Profs"
```
Comment (Jason-Ku) [inline Entity/Profs.py:9]: We need an id column to keep the pattern we have in other entities so insert/update_entity will still work for this Entity

File: Entity/ProfessorSectionView.py [hunk 3]
```diff
@@ -16,10 +16,9 @@ class SectionType(enum.Enum):
 
 class ProfessorSectionView(Base):
     __tablename__ = "Professor_Teaches_Section"
-    id = Column(Integer, primary_key=True)
     id_sections = Column(Integer)
```
Comment (Jason-Ku) [inline Entity/ProfessorSectionView.py:19]: Also need ID column for this entity

**Comment by mfekadu** [comment]
Discussion

@Jason-Ku 

1. **View** `entity_type` should not be acceptable input to `insert_entity` because they are generated by the MySQL DBMS

2. we might include a property `_is_view` to any **Entity** that is a **View** rather than a **Table**

3. a well-documented error when **View** input  to `insert_entity` would be cool

**Comment by mfekadu** [comment]
#143

**Pull request merged**